### PR TITLE
Translate 4 terms to Italian

### PR DESCRIPTION
**Pull request by ftabaro**
## Author: 

- Francesco Tabaro

## Language: 

- it

## Terms defined:

- abandonware
- absolute file path
- absolute row number
- abstract method

### CODE REVIEW

**Comment by lucaferranti** [comment]
Hi @ftabaro , I'll review this in the upcoming days. Meanwhile, I noticed the PR has several noise changes (remove trailing spaces, empty lines, introduce new line breaks). I presume these were introduced because of your autoformatter?

Could you undo those changes, so that the PR effectively only adds the 4 Italian terms?

**Comment by lucaferranti** [comment]
a way of undoing the extra changes:
- save the definitions you had added somewhere
- reset to HEAD~4
- turn off your autoformatter
- reinsert it your changes and force push to this branch

**Comment by elletjies** [comment]
@lucaferranti just checking in to see what is still needed for this PR?

**Comment by lucaferranti** [comment]
@elletjies the problem I pointed out before seems to persist. If you look at the changes, this PR does also a lot of formatting across the whole file, I would guess because of an automatic formatter the PR author has.

I think these extra changes not related to the 4 Italian terms should be removed before merging. This is not only because it makes review much easier, but also because merging the PR as such would probably lead a lot of merge conflicts in other PRs.

**Comment by elletjies** [comment]
@ftabaro just bringing to your attention the changes reviewers have suggested. Please let us know if we can support you.

**Comment by ftabaro** [comment]
Done. Still unclear how to turn off auto-formatting in VSCode. I have no plugin installed and couldn't figure out the setting. Sorry for the nuisance. Cheers.

File: glossary.yml [hunk 1]
```diff
@@ -137,6 +137,10 @@
     term: "Matlotla"
     def: >
       Serweboleta (software) se se sa tlholeng se tlhokomelwa kgotsa se tlhabololwa.
+  it:
+    term: "abandonware"
+    def: >
+      "Un software abbandonato, non più mantenuto"
```
Comment (lucaferranti) [inline glossary.yml:143]:
```suggestion
      "Un software abbandonato, non più mantenuto."
```

**Comment by elletjies** [comment]
@ftabaro Thank you for contributing to Glosario in 2022! We are looking forward to receiving more PRs from you during 2023!

**Comment by elletjies** [comment]
@all-contributors please add @ftabaro for translation

**Pull request merged**